include/deal.II/matrix_free/matrix_free.h

```diff
@@ -1912,6 +1917,11 @@ class MatrixFree : public Subscriptor
    * run between the given values in the field
    * AdditionalData::cell_vectorization_category for non-hp-DoFHandler types
    * and return the active FE index in the hp-adaptive case.
+   *
+   * @note In the non-hp-DoFHandler case, a categroy of a cell batch is given
+   * as the maximum category of any of its cell. In the hp case or the case that
+   * MatrixFree::AdditionalData::cell_vectorization_categories_strict was
+   * enabled, it is guaranteed that all cells have the same categroy.
```

Comment:
You should run a spellchecker on this particular word. :wink:
```suggestion
   * enabled, it is guaranteed that all cells have the same category.
```